develop/start-develop.sh

```diff
@@ -3,5 +3,5 @@ set -x
 set -e
 cd $SOURCE_DIR
 npm install
-node_modules/bower/bin/bower install --allow-root
-node_modules/grunt-cli/bin/grunt serve --allow-root
+bower install --allow-root
```

Comment:
This did not work on my local machine due to several messages about dependencies not being found. In one case ``q`` could not be found, in another case a dependency relating to ``grunt``. Installing globally solved all problems, presumably because it resolved some environmental issues allowing the dependencies to be found.